### DISCUSSION TITLE
Google home integrazione GCP

### DISCUSSION THREAD

**Discussion by manustars** [indomus/forum#829] [HUB: Home Assistant]
Salve a tutti, premetto che ho seguito la guida passo passo e sembra funzionare, ma gia durante l'implementazione ho avuto seri problemi nella parte di test.

Quando vado a fare il test mi esce sempre questa schermata [https://ibb.co/N15hz39](url), ma funziona l'integrazioe, a questo punto non so se è ancora valida l'azione di fare il test per tenerla attiva ogni mese ma oggi mi ritrovo che dall'app google assistant non posso fare il relik del progetto (mi da come se il link non esiste più).

Ora posso rifare l'intera procedura non è un problema, ma tutti i vecchi progetti che ho eleminati li posso togliere dall'integrazione?

**Comment by snakuzzo** [comment]
Il test non serve più

**Reply by manustars** [reply]
Perfetto
